bin/publish

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env lsc
+
+require! {
+  'chalk' : {bold, red, cyan, yellow, green}
+  'inquirer'
+  'replace-in-file' : replace
+  'semver'
+  'shelljs/global'
+}
+
+SUBPROJECTS_TO_PUBLISH = <[ cli
+                            exo-add
+                            exo-clone
+                            exo-create
+                            exo-deploy
+                            exo-lint
+                            exo-run
+                            exo-setup
+                            exo-sync
+                            exo-test
+                            exosphere-shared
+                            website          ]>
```

Comment:
Tests for `website` still do not pass at this point, so technically this publish script won't finish.